### ISSUE TITLE
Connection error to a host other than localhost

### ISSUE THREAD

**Issue by afls** [JabRef/jabref#3118]
JabRef version 3.8.2 on Windows 7 (64 bit)

Hello,
using "Connect to shared database" to connect to MySQL on a host that is not localhost results to a "Connection error" message: "Access denied for user 'db@local-PC-name.de' (using password:YES)". 

The menu "Connect to shared database" does not use the host name (server name) entered into the text field but always uses the local PC name. The same error message also occurs then entering not the host name but the IP of the server.

I hope that anybody can help or fix the problem, since using a shared database only at localhost does not make sense in a team with many users.

Regards, afls

**Comment by koppor** [comment]
I usually tried postgres.

@stefan-kolb Can you quickly recommend me a free PaaS MySQL service to test?

**Comment by stefan-kolb** [comment]
This should not be related to MySQL, should it?

**Comment by stefan-kolb** [comment]
@koppor http://w2.cleardb.net/

**Comment by koppor** [comment]
@memering seems to be able to connect to a remote database. See https://github.com/JabRef/jabref/issues/3736.

**Comment by stefan-kolb** [comment]
Thank you for your report :+1:
However, I cannot reproduce this behavior on the current `master`. Can you please try the latest build from http://builds.jabref.org/master.

**Comment by memering** [comment]
It's an SSH tunnel in my case, so from JabRef's point of view it's localhost.

**Comment by stefan-kolb** [comment]
Cannot test it with Heroku Postgres as we don't support SSL #3419

**Comment by stefan-kolb** [comment]
Ok, i can connect to a JawsDB remote DB. However, I can hardly edit information as every step nearly freezes the UI. Is every change sent to the database, e.g., every key pressed?

**Comment by stefan-kolb** [comment]
In general this is functional but kind of unusable as every single event synchronizes with the database.

**Comment by koppor** [comment]
This design decision was per intention (somehow). It is event-triggered. I think, @lenhard once made an improvement at the event that only after a few character changes the event is fired. Maybe, we should update that DB sync code that only after a field is changed, the event is sent?

**Comment by stefan-kolb** [comment]
@koppor this problem is tracked in a separate issue #3736
